get-latest-tgf.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 get_local_tgf_version () {
-    TGF_LOCAL_VERSION=$(tgf --current-version | awk -F\  '{print $2}')
+    TGF_LOCAL_VERSION=$($TGF_PATH/tgf --current-version | awk -F\  '{print $2}')
```

Comment:
TGF_PATH is not set here. This causes the script to update TGF everytime

Reply:
I would set TGF_PATH at the beginning completely (right after the shebang)

Reply:
```suggestion
    TGF_LOCAL_VERSION=$($TGF --current-version | awk -F\  '{print $2}')
```